core/test/lib/dependency-graph/simulator/network-analyzer-test.js

```diff
@@ -202,6 +202,16 @@ describe('DependencyGraph/Simulator/NetworkAnalyzer', () => {
       assert.deepStrictEqual(result.get('https://example.com'), expected);
     });
 
+    it('should use coarse estimates on a per-origin basis', () => {
+      const records = [
+        createRecord({url: 'https://example.com', timing: {connectStart: 1, connectEnd: 100, sendStart: 150}}),
+        createRecord({url: 'https://example2.com', timing: {sendStart: 150}}),
+      ];
+      const result = NetworkAnalyzer.estimateRTTByOrigin(records);
+      assert.deepStrictEqual(result.get('https://example.com'), {min: 99, max: 99, avg: 99, median: 99});
+      assert.deepStrictEqual(result.get('https://example2.com'), {min: 15, max: 15, avg: 15, median: 15});
```

Comment:
Without these changes, `result.get('https://example2.com/')` is undefined.

Reply:
is that better or worse than `DEFAULT_SERVER_RESPONSE_TIME = 30`? It's not clear from this test if it's a good estimate or not

Reply:
Whether it's a better estimate or not seems besides the point at this level - the test data is all made up. But the larger point of "is this actually better" is clear. Maybe the coarse estimations are just totally bonkers and we were better off when we skipped it.

15 is the result of `sendStart / 3 roundtrips = 50; 50 * 0.3 fudge factor = 15`. There are no notes on how the 0.3 fudge factor is derived, but that's the one knob we have here. We should probably do some analysis of tons of records and see how known-rtt (when SSL and Connect timings are present) correlates with the various strategies used for coarse estimations. Is that worth the effort?